not-paid.js

```diff
@@ -3,6 +3,7 @@
 	/* change these variables as you wish */
 	var due_date = new Date('2017-02-27');
 	var days_deadline = 60;
+    var deadline_message = ""; /* Leave empty string to disable */
```

Comment:
This indentation is also off, I believe.

```suggestion
        var deadline_message = ""; /* Leave empty string to disable */
```

Reply:
Yes, it seems that way, @sam3d.  Thanks!